scripts/cli

```diff
@@ -28,7 +28,7 @@ cmd_typecheck () {
 
 cmd_lint () {
   local project_path="${1}"
-  eslint "$project_path"
+  eslint "$project_path" 
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏not sure to see what is going on here 🤔 involuntary update?

Reply:
Nice catch.
Yes it's an involuntary update from when I set `--fix`